server/tests/unit/address.test.js

```diff
@@ -0,0 +1,142 @@
+        country: faker.address.countryCode(),
+      })
+      .expect(400);
+  });
+  test("address form should return 400 for invalid phone", async () => {
+    request(router)
+      .post("/")
+      .send({
+        name:  faker.name.findName(),
+        email: faker.internet.email(),
+        phone: "",
+        street: faker.address.streetAddress(),
+        city: faker.address.city(),
+        state: faker.address.state(),
+        pin_code: faker.address.zipCode(),
+        country: faker.address.countryCode(),
+      })
+      .expect(400);
+  });
+  test("address form should return 400 for invalid street", async () => {
+    request(router)
+      .post("/")
+      .send({
+        name:  faker.name.findName(),
+        email: faker.internet.email(),
+        phone: faker.phone.phoneNumber("991#######"),
+        street: "",
+        city: faker.address.city(),
+        state: faker.address.state(),
+        pin_code: faker.address.zipCode(),
+        country: faker.address.countryCode(),
+      })
+      .expect(400);
+  });
+  test("address form should return 400 for invalid city", async () => {
+    request(router)
+      .post("/")
+      .send({
+        name:  faker.name.findName(),
+        email: faker.internet.email(),
+        phone: faker.phone.phoneNumber("991#######"),
+        street: faker.address.streetAddress(),
+        city: "",
+        state: faker.address.state(),
+        pin_code: faker.address.zipCode(),
+        country: faker.address.countryCode(),
+      })
+      .expect(400);
+  });
+
+  test("address form should return 400 for invalid pin_code", async () => {
+    request(router)
+      .post("/")
+      .send({
+        name:  faker.name.findName(),
+        email: faker.internet.email(),
+        phone: faker.phone.phoneNumber("991#######"),
+        street: faker.address.streetAddress(),
+        city: faker.address.city(),
+        state: faker.address.state(),
+        pin_code:"",
+        country: faker.address.countryCode(),
+
+      })
+      .expect(400);
+  });
+
+  test("address form should return 400 for invalid state", async () => {
+    request(router)
+      .post("/")
+      .send({
+        name:  faker.name.findName(),
+        email: faker.internet.email(),
+        phone: faker.phone.phoneNumber("991#######"),
+        street: faker.address.streetAddress(),
+        city: faker.address.city(),
+        state: "",
+        pin_code: faker.address.zipCode(),
+        country: faker.address.countryCode(),
+
+      })
+      .expect(400);
+  });
+
+  test("address form should return 400 for invalid country", async () => {
+    request(router)
+      .post("/")
+      .send({
+        name:  faker.name.findName(),
+        email: faker.internet.email(),
+        phone: faker.phone.phoneNumber("991#######"),
+        street: faker.address.streetAddress(),
+        city: faker.address.city(),
+        state: faker.address.state(),
+        pin_code: faker.address.zipCode(),
```

Comment:
why are you duplicating some of these lines in the other methods in this file. DRY